generator/internal/parser/protobuf.go

```diff
@@ -354,7 +354,7 @@ func makeAPIForProtobuf(serviceConfig *serviceconfig.Service, req *pluginpb.Code
 		result.Name = strings.TrimSuffix(serviceConfig.Name, ".googleapis.com")
 	}
 	updateMethodPagination(result)
-	return result
+	return result, result.Accept(&api.InitializationVisitor{})
```

Comment:
I think this is too early to call `Accept`.  The codecs also introduce the well-known types:

https://github.com/googleapis/google-cloud-rust/blob/d1348119749a52e9bdc48267eca3d1746d330250/generator/internal/language/rust.go#L274-L278

And that happens much later:

https://github.com/googleapis/google-cloud-rust/blob/d1348119749a52e9bdc48267eca3d1746d330250/generator/internal/language/rusttemplate.go#L170

I think you want to run this visitor after the well-known types are loaded.

Reply:
Moved. I think we could refactor this later, to avoid repetition. For now I am running the visitor in both language templates (not worse than before though, as I was running it in two parsers).